django/contrib/admin/templatetags/admin_list.py

```diff
@@ -24,64 +24,31 @@
 
 register = Library()
 
-DOT = '.'
-
 
 @register.simple_tag
 def paginator_number(cl, i):
     """
     Generate an individual page index link in a paginated list.
     """
-    if i == DOT:
-        return '… '
+    if i == cl.paginator.ELLIPSIS:
```

Comment:
Should we call it "ellipsis"? It's basically an option to customize a separator and do not use ellipsis :thinking:

Reply:
Yeah. So I changed it to `ELLIPSIS` from `DOT` as the intent is to make it easy to use by having a standard character for the elided sections of the page range that can just be pushed out for display purposes. It has also been made translatable for that reason, although another language may not use an ellipsis.

I'm struggling to think of a better name for this. Perhaps `ELIDE`, `SKIP`, `OMIT`, `BREAK`, or `CUT`?

Reply:
Let's go with "ellipsis": it's primary meaning is _the omision_, for which folks _could_ use any symbol. It's secondary meaning is the three dots `…` that we actually use.